pages/common/nping.md

```diff
@@ -0,0 +1,36 @@
+# nping
+
+> Network packet generation tool / ping utility.
+> More information: <https://nmap.org/nping/>.
+
+- ICMP ping if the user is allowed to, otherwise TCP ping to a specific host:
+
+`nping {{example.com}}`
+
+- ICMP ping to a specific host assuming that the user is allowed to do so:
+
+`nping --icmp --privileged {{example.com}}`
+
+- UDP ping to a specific host:
+
+`nping --udp {{example.com}}`
+
+- TCP ping on a specific port:
+
+`nping --tcp --dest-port {{443}} {{example.com}}`
+
+- Ping a certain number:
+
+`nping --count {{10}} {{example.com}}`
+
+- Wait a certain time between each ping:
+
+`nping --delay {{5s}} {{example.com}}`
+
+- Send the request over a specified interface:
```

Comment:
```suggestion
- Send the request over the specified interface:
```

Reply:
Why "the" and not "a"? "the" refers to a specific thing, doesn't it? And "a" would be an indefinite thing and here it is irrelevant which host is pinged.

In the case of `nc` or `ping`, "a" rather than "the" was used, wasn't it?

Reply:
"the specifi***ed*** host" refers to a specific host (the one that the user has specified). That's why "the" should be used.

`nc`:
> Listen on ***a specified*** TCP port and print any data received

This example is the same, it refers to a specific thing. That's why the definite article should be used. "the specified" sounds a lot better to me, but maybe it's just me.

`ping`:
> Ping a host only ***a specific*** number of times

Here, "a specific" refers to any host that could be specified by the user, hence the "a".

So, you should either use "a specific" or "the specified". Or at least that's how I see it, I may be wrong. Let's wait for other maintainers.

---

"the specified" is more frequently used though. I ran `rg` on the tldr repository:
```bash
$ rg -l " a specified " | wc -l
151

$ rg -l " the specified " | wc -l
282
```
And "a specific" is even more frequent:
```bash
$ rg -l " a specific " | wc -l
1021
```

Reply:
We do use both `the` and `a` along with specific/specified in tldr. Here, I think `a` would align better with other examples.